quickstarts/library-process/library/src/main/java/org/openshift/quickstarts/rhpam/kieserver/library/Library.java

```diff
@@ -17,15 +17,16 @@
 
 public final class Library {
 
-    private static final Object[][] INIT = new Object[][] {
-        new Object[] {"978-0-307-35193-7", "World War Z", "An Oral History of the Zombie War", 1},
-        new Object[] {"978-0-7360-9829-8", "Successful Sports Officiating", "American Sports Education Program.", 3},
-        new Object[] {"978-0-7434-8773-3", "The Time Machine", "H. G. Wells' story of a time traveler.", 8},
-        new Object[] {"978-1-101-15402-1", "The Island of Dr. Moreau", "H. G. Wells' story of what may be most relevant to modern ethical dimemmas.", 5},
-        new Object[] {"978-1-4000-5-80-2", "The Zombie Survival Guide", "Complete Protection from the Living Dead", 6},
-        new Object[] {"978-1-448-14153-1", "Doctor Who: Summer Falls", "Story of Amelia Williams.", 2},
-        new Object[] {"978-1-4516-7486-6", "Tesla, Man Out of Time", "Explores the brilliant and prescient mind of one of the twentieth century's greatest scientists and inventors, Nikola Tesla.", 4},
-        new Object[] {"978-1-59474-449-5", "Pride and Prejudice and Zombies", "The Classic Regency Romance -- Now with Ultraviolent Zombie Mayhem!", 7}
+    private static final Object[][] INIT = new Object[][]{
+                                                          new Object[]{"978-0-307-35193-7", "World War Z", "An Oral History of the Zombie War", 1},
```

Comment:
Why introduce such a huge indent? Are you using spaces (preferred) or tabs?

Reply:
See above.

Reply:
Same. :)